Sources/SCTE35/HelperConverter/BitConverter.swift

```diff
@@ -67,7 +146,7 @@ class BitConverter {
         return string
     }
 
-    static func smpteString(fromBits bits: [Bit]) -> String? {
+    static func umidString(fromBits bits: [Bit]) -> String? {
```

Comment:
Small nit:  The correct name is `umid`, `smpte` is the name of the society that maintains the standard.